src/lib.rs

```diff
@@ -149,6 +149,39 @@ pub fn list_devices() -> Result<impl Iterator<Item = DeviceInfo>, Error> {
     platform::list_devices()
 }
 
+/// Get an iterator listing the system USB buses.
+///
+/// ### Examples
+///
+/// Filter for a specific bus ID:
+///
+/// ```no_run
+/// use nusb;
+/// let hub = nusb::list_buses().unwrap()
+///    .find(|bus| bus.bus_id().parse() == Ok(1))
```

Comment:
Should probably not encourage parsing the bus ID as int, since it's only an int on Linux.